### PR TITLE
Newsletter 10: beehive, rustsim, safearch, svgface

### PR DESCRIPTION
**Pull request by ozkriff**
Part of #139 (_"Newsletter 10: Coordination/Tracking"_) 

@Lokathor @sebcrozet @dabreegster @toasteater r?

### CODE REVIEW

**Comment by ozkriff** [comment]
Merging in a few hours if there're no objections

**Comment by Lokathor** [comment]
looks good in the `safe_arch` portion.

**Comment by dabreegster** [comment]
LGTM for `svg_face`

**Comment by sebcrozet** [comment]
No objection for the rustsim survey 👍

File: content/posts/newsletter-010/index.md [hunk 1]
```diff
@@ -529,6 +556,53 @@ _Discussions: [/r/rust](https://www.reddit.com/r/rust/comments/gf42ip/kas_gui_04
...
+for working with 3D hexagonal maps.
+It's based on [Red Blob Games's guide][rbg-guide].
+
+[beehive]: https://github.com/toasteater/beehive
+[@toast_dev]: https://twitter.com/toast_dev
+[rbg-guide]: https://redblobgames.com/grids/hexagons
+
+### [RustSim Survey][rustsim-survey]
+
+![rustim logo: lab flask in a gear](rustsim.png)]
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-010/index.md:579]: Stray square bracket here, rest of it LGTM!

```suggestion
![rustim logo: lab flask in a gear](rustsim.png)
```

**Pull request merged**